addons/delivery/views/delivery_view.xml

```diff
@@ -87,6 +86,7 @@
                                 </div>
                             </button>
                         </div>
+                        <widget name="web_ribbon" title="Archived" bg_color="bg-danger" attrs="{'invisible': [('active', '=', True)]}"/>
```

Comment:
using title instead of text is good, but I'm not sure that it should be moved

![image](https://user-images.githubusercontent.com/186131/115869909-d695c800-a43e-11eb-95bf-28efb03c5834.png)